src/measure_theory/vector_measure.lean

```diff
@@ -618,6 +630,182 @@ begin
   { rw [v.not_measurable hi, w.not_measurable hi] }
 end
 
+notation v ` ≤[`:50 i:50 `] `:0 w:50 :=
```

Comment:
can you maybe put this notation in a locale `measure_theory` (like for the Hausdorff measure), to avoid clashes with other areas of maths that might want the same kind of notation?

Reply:
when I change that line to
```
localized "notation v ` ≤[`:50 i:50 `] `:0 w:50 :=
measure_theory.vector_measure.restrict v i ≤ measure_theory.vector_measure.restrict w i"
in measure_theory
```
the notation doesn't work if there is more than 1 character within the square brackets (I think lean thinks its a list in that case). Is there a way to fix this?

Reply:
I don't think you're supposed to be changing the precedence of the `]` token, since that's already set by lots of other notation. I assume your problem isn't something basic like forgetting to do `open_locale`?

Can you compare the output of `print notation` before and after switching to localized?

Reply:
print notation outputs the same before and after:
```
#print notation ≤[]
/- 
before localized:
_ `≤[`:50 _:50 `]`:0 _:50 := has_le.le (measure_theory.vector_measure.restrict #2 #1) (measure_theory.vector_measure.restrict #0 #1)
after localized: 
_ `≤[`:50 _:50 `]`:0 _:50 := has_le.le (measure_theory.vector_measure.restrict #2 #1) (measure_theory.vector_measure.restrict #0 #1)
-/
```
I tried removing the precedence but that didn't fixed it either (same error same place)

Reply:
I think it's just because you declared this in a section. After the section, the notation is not open anymore. So you can either declare the notation outside of a section (for instance on line 607), or `open_locale measure_theory` at the end of the section where you declare the notation, to make sure it is available from this point on.

Reply:
Thanks!